### PR TITLE
fix ConcurrentModificationException in case of multiple projects, closes #26

### LINKED ISSUE #26: CME in build with multiple projects
missinglink is not thread-safe so it fails or produces inconsistent results if you have run it on more than one project.
Please see https://github.com/OlegYch/sbt-missinglink/commit/f0e0c09b37a9b65ded44fa71b4c2c1ffa73dc754 for a fix.

### PR DESCRIPTION
**Pull request by OlegYch**

### CODE REVIEW

File: src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala [hunk 1]
```diff
@@ -4,11 +4,9 @@ import sbt._
 import sbt.Keys._
 import sbt.librarymanagement.ModuleFilter
 import sbt.plugins.JvmPlugin
-
 import java.io.FileInputStream
 
 import scala.collection.JavaConverters._
-
 import com.spotify.missinglink.{ArtifactLoader, Conflict, ConflictChecker, Java9ModuleLoader}
```
Comment (sjrd) [inline src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala]: Could you revert the unrelated blank line changes here, please?

File: src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala [hunk 2]
```diff
@@ -53,49 +51,55 @@ object MissingLinkPlugin extends AutoPlugin {
   override def requires: Plugins = JvmPlugin
   override def trigger: PluginTrigger = allRequirements
 
+  //missinglink-core has non-threadsafe caches
+  val missinglinkConflictsTag = Tags.Tag("missinglinkConflicts")
   val configSettings: Seq[Setting[_]] = Def.settings(
```
Comment (sjrd) [inline src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala]: And here, could you ass a blank line between those two lines. (We also have a blank line between two class members as soon as one of them spans several lines (which is the case of `configSettings`).

**Comment by OlegYch** [comment]
sorry,  i'll have to leave the pleasure to someone else

**Comment by sjrd** [comment]
Sure, no worries. I took care of it in #29. Thanks a lot for the contribution!

And I'm sorry if my comments came out the wrong way.
